Sources/MapboxNavigation/ExitView.swift

```diff
@@ -23,6 +23,15 @@ public class ExitView: StylableView {
         }
     }
     
+    @objc public dynamic var highlightColor: UIColor? {
```

Comment:
Because `highlightColor` will only apply when the `isHighlighted` is `true`. And it will apply to the `borderColor` and `foregroundColor` directly.